packs/csi_openstack_cinder/templates/csi_openstack_cinder.nomad.tpl

```diff
@@ -0,0 +1,84 @@
+job [[ template "job_name" . ]] {
+  [[ template "region" . ]]
+
+  datacenters = [[ .csi_openstack_cinder.datacenters | toPrettyJson ]]
+
+  type = system
```

Comment:
```suggestion
  type = "system"
```

This needs to be quoted, or the template engine thinks it's an uninterpolated variable and Nomad shows the following error:

```
    2021-11-05T18:12:58.655Z [ERROR] http: request failed: method=POST path="/v1/jobs?namespace=default&region=global"
  error=
  | 1 error occurred:
  |     * Invalid job type: "${system}"
  |
   code=500
```

Reply:
Don't know how I overlooked that one lol. All fixed up.